src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -100,6 +100,7 @@ ExecNetwork::ExecNetwork(const InferenceEngine::CNNNetwork &network,
     } else {
         auto streamsExecutorConfig = InferenceEngine::IStreamsExecutor::Config::MakeDefaultMultiThreaded(_cfg.streamExecutorConfig, isFloatModel);
         streamsExecutorConfig._name = "CPUStreamsExecutor";
+        _cfg.streamExecutorConfig._threads = streamsExecutorConfig._threadsPerStream * streamsExecutorConfig._streams;
```

Comment:
would _threads and _streams contain both Pcore and Ecore parts?

Reply:
No, I added Hybrid aware calculating.